pkg/epforwarder/epforwarder.go

```diff
@@ -69,6 +70,18 @@ var passthroughPipelineDescs = []passthroughPipelineDesc{
 		defaultBatchMaxSize:           pkgconfig.DefaultBatchMaxSize,
 		defaultInputChanSize:          pkgconfig.DefaultInputChanSize,
 	},
+	{
+		eventType:              eventTypeDBMMetadata,
+		contentType:            http.JSONContentType,
+		endpointsConfigPrefix:  "database_monitoring.metadata.",
+		hostnameEndpointPrefix: "dbm-metrics-intake.",
+		intakeTrackType:        "dbmmetadata",
+		// raise the default batch_max_concurrent_send from 0 to 10 to ensure this pipeline is able to handle 4k events/s
+		defaultBatchMaxConcurrentSend: 10,
+		defaultBatchMaxContentSize:    20e6,
```

Comment:
Is this needed for metadata?

Reply:
we will ingest database schemas, which could end up being a lot of data